### PR TITLE
Create answers.md

### PR DESCRIPTION
**Pull request by aryanm1997**
answers

### CODE REVIEW

File: answers.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+
+
+
+1)python 3.7
+
+2)def convert(list):
```
Comment (niksmac) [inline answers.md:6]: The question is to Write a function that takes a number and returns a list of its digits in an array. 
- It accepts a number like `43545` and it should return `[4, 3, 5, 4, 5]`

File: answers.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    s = [str(i) for i in list]
+    res = int("".join(s))
+    return(res)
+list = [1, 2, 3]
+print(convert(list))
+
+
+
+3)str1=input("Enter the string:")
```
Comment (niksmac) [inline answers.md:15]: The question is to Remove duplicates of an array and returning an array of only unique elements 

- It accepts an array like `[4, 3, 5, 4, 5]` and it should return `[4, 3, 5]`

**Review by niksmac — CHANGES_REQUESTED** [review]
Please fix the issues and submit again.

File: answers.md (hunk 2 above)
Comment (aryanm1997) [reply answers.md:15]:
```suggestion
3)my_list = [4,3,5,4,5]
my_final_list = set(my_list)
print(list(my_final_list))
```

File: answers.md (hunk 1 above)
Comment (aryanm1997) [reply answers.md:6]: 2)test_list = [4,3,5,4,5]
print ("The original list is : " +  str(test_list))
res = []
for i in test_list:
    if i not in res:
        res.append(i) 
print ("The list after removing duplicates : " + str(res))

**Pull request closed without merging**